editor/debugger/script_editor_debugger.cpp

```diff
@@ -884,6 +884,7 @@ void ScriptEditorDebugger::_clear_breakpoints() {
 }
 
 void ScriptEditorDebugger::start(Ref<RemoteDebuggerPeer> p_peer) {
+	error_tree->clear();
 	error_count = 0;
 	warning_count = 0;
```

Comment:
Maybe call `_clear_errors_list()` instead?

Any idea when the regression started?

Reply:
> Any idea when the regression started?

Seems the previous `error_tree->clear();` present in `3.x` was removed in cbc450c0e53dbfc31a30f5ebf37c51cee77cde5c, CC @Faless.

The fix in this PR seems fine, makes sense to me to clear on start.

Reply:
Yeah, but since `_clear_errors_list()` runs a process loop at the end, I didn't use it to prevent any side-effects. I've tested with it now and seems to run fine, but I'll wait on Faless's input before pushing.

Reply:
`_clear_errors_list()` is just this, so it's the same code no?
https://github.com/Ev1lbl0w/godot/blob/fix_debug_errors_not_clearing/editor/debugger/script_editor_debugger.cpp#L1401-L1405

Reply:
Oh yeah, you're right; it changed after the refactoring and I didn't even bother to check it again :sweat_smile:

Yup it's safe then, so I'll change it